noetic/distribution.yaml

```diff
@@ -2399,6 +2399,21 @@ repositories:
       url: https://github.com/ros/rospack.git
       version: noetic-devel
     status: maintained
+  rosparam_shortcuts:
+    doc:
+      type: git
+      url: https://github.com/davetcoleman/rosparam_shortcuts.git
+      version: noetic-devel
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/PickNikRobotics/rosparam_shortcuts-release.git
+      version: 0.4.0-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/PickNikRobotics/rosparam_shortcuts.git
```

Comment:
I'm surprised the `doc` and `source` sections have different URLs. Is that intended? If not, which is correct: `davetcoleman` or `PickNicRobotics`?

Reply:
Thanks @sloretz for catching it, I didn't intend to have them different but seem like it's ok since melodic does have the same URLs for both https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml#L9343

Reply:
It's certainly allowed, but its not recommended.  I'd suggest making it consistent here, and then updating Melodic to match.